service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/OrderDocumentGeneratorHolder.java

```diff
@@ -58,7 +60,8 @@ public Map<Order, DocmosisParameterGenerator> getTypeToGenerator() {
                 c43ChildArrangementOrderDocumentParameterGenerator,
                 c43ASpecialGuardianshipOrderDocumentParameterGenerator,
                 c47AParameterGenerator,
-                c45aParentalResponsibilityOrderDocumentParameterGenerator
```

Comment:
yeah this'll be a pain for merging so might be good to get an answer from the BA's about which ticket will be merged in first